superset/assets/src/components/OnPasteSelect.jsx

```diff
@@ -64,6 +64,23 @@ export default class OnPasteSelect extends React.Component {
       }
     }
   }
+  customFilter(option, searchText){
+    // customFilter is passed as a funciton to react-select library 
+    // in the case that there is a column name and verbose name 
+    if( !option.column_name){
```

Comment:
looks like styling is off here, you should run linters locally to style it correctly

Reply:
fixed